apps/api/src/evm/config.ts

```diff
@@ -7,7 +7,16 @@ import ProxyFactory from './abis/ProxyFactory.json';
 import SimpleQuorumTimelockExecutionStrategy from './abis/SimpleQuorumTimelockExecutionStrategy.json';
 import Space from './abis/Space.json';
 
-type NetworkID = 'eth' | 'sep' | 'oeth' | 'matic' | 'arb1' | 'base' | 'mnt';
+type NetworkID =
+  | 'eth'
+  | 'sep'
+  | 'oeth'
+  | 'matic'
+  | 'arb1'
+  | 'base'
+  | 'mnt'
+  | 'apechain'
```

Comment:
We should use `ape` instead. This is what they are using for Safe https://safe.apechain.com/home?safe=ape:0xD60c1681Ac2f3DD4Cdc0a8F35b56DdC75d28bcd5

```suggestion
  | 'ape'
```